ios/RNModules/RNAccountManager/RNAccountManager/RNAccountManager.m

```diff
@@ -6,12 +6,19 @@
 //  Copyright © 2019 Kiwi.com. All rights reserved.
 //
 
+#import "RNAccountManager.h"
+
 const NSNotificationName RNKiwiAccountDeletedNotification = @"RNKiwiAccountDeletedNotification";
 
-#import "RNKiwiAccountUpdater.h"
 
-@implementation RNKiwiAccountUpdater
 
+@implementation RNAccountManager
+
+RCT_EXPORT_MODULE();
+
++ (BOOL)requiresMainQueueSetup {
```

Comment:
On the native side, this is doing nothing here that requires the main queue - why was this added?

Reply:
I can try running without it, I really just saw that most of our modules have this

Reply:
Seems it was not necessary, so I removed it again. Thanks.